utils/iio_info.c

```diff
@@ -33,12 +34,16 @@
 static struct iio_context *ctx;
 
 static const struct option options[] = {
-	{0, 0, 0, 0},
+	{ "read-debugfs", no_argument, NULL, 'd' },
```

Comment:
When `iio_info` generates its output, it categorizes the attributes into context attributes, device attributes, buffer attributes, channel attributes, and debug attributes. Therefore, it makes sense to use the same naming. I suggest:
`{ "read-debug-attr", no_argument, NULL, 'd' }`